.eslintrc.yml

```diff
@@ -13,12 +13,16 @@ rules:
   no-console: "off"
   import/order: [error, newlines-between: always]
   import-order-alphabetical/order: [error]
+  import/no-unresolved:
+    - error
+    - commonjs: true
+      ignore: [ ^aws-sdk$, ^oracledb$, ^request-promise-native$, contrib/ ]
 overrides:
-  - files:
-      - "**/*-example.js"
-      - src/api/v1/db/awsS3/aws-operations.js
-    rules:
-      import/no-unresolved: "off"
+  files:
+    - src/api/v1/db/oracledb/pets-dao-example.js
```

Comment:
Why is this changed? I think all of the `example.js` should be applied for these rules, right?

Reply:
Should be fixed now